OpenSimRT/RealTime/experimental/RealTimeAnalysisExtended/tests/TestRTExtFromFile.cpp

```diff
@@ -0,0 +1,320 @@
+    auto grfLeftPointExpressed =
+            ini.getString(section, "GRF_LEFT_POINT_EXPRESSED_IN_BODY", "");
+    auto grfLeftPointIdentifier =
+            ini.getString(section, "GRF_LEFT_POINT_IDENTIFIER", "");
+    auto grfLeftForceIdentifier =
+            ini.getString(section, "GRF_LEFT_FORCE_IDENTIFIER", "");
+    auto grfLeftTorqueIdentifier =
+            ini.getString(section, "GRF_LEFT_TORQUE_IDENTIFIER", "");
+
+    // Low pass filter parameters
+    auto memory = ini.getInteger(section, "MEMORY", 0);
+    auto cutoffFreq = ini.getReal(section, "CUTOFF_FREQ", 0);
+    auto delay = ini.getInteger(section, "DELAY", 0);
+    auto splineOrder = ini.getInteger(section, "SPLINE_ORDER", 0);
+    auto calcDer = ini.getBoolean(section, "CALC_DER", true);
+
+    // contact-force-based detector parameters
+    auto windowSize = ini.getInteger(section, "WINDOW_SIZE", 0);
+    auto threshold = ini.getReal(section, "THRESHOLD", 0);
+    auto platform_offset = ini.getReal(section, "PLATFORM_OFFSET", 0.0);
+    auto rFootBodyName = ini.getString(section, "RIGHT_FOOT_BODY_NAME", "");
+    auto lFootBodyName = ini.getString(section, "LEFT_FOOT_BODY_NAME", "");
+    auto rHeelSphereLocation =
+            ini.getSimtkVec(section, "RIGHT_HEEL_SPHERE_LOCATION", Vec3(0));
+    auto lHeelSphereLocation =
+            ini.getSimtkVec(section, "LEFT_HEEL_SPHERE_LOCATION", Vec3(0));
+    auto rToeSphereLocation =
+            ini.getSimtkVec(section, "RIGHT_TOE_SPHERE_LOCATION", Vec3(0));
+    auto lToeSphereLocation =
+            ini.getSimtkVec(section, "LEFT_TOE_SPHERE_LOCATION", Vec3(0));
+    auto contactSphereRadius = ini.getReal(section, "SPHERE_RADIUS", 0);
+
+    // grfm parameters
+    bool useGRFMPrediction =
+            ini.getBoolean(section, "USE_GRFM_PREDICTION", true);
+    auto grfmMethod = ini.getString(section, "METHOD", "");
+    auto pelvisBodyName = ini.getString(section, "PELVIS_BODY_NAME", "");
+    auto rHeelCoPLocation =
+            ini.getSimtkVec(section, "RIGHT_HEEL_STATION_LOCATION", Vec3(0));
+    auto lHeelCoPLocation =
+            ini.getSimtkVec(section, "LEFT_HEEL_STATION_LOCATION", Vec3(0));
+    auto rToeCoPLocation =
+            ini.getSimtkVec(section, "RIGHT_TOE_STATION_LOCATION", Vec3(0));
+    auto lToeCoPLocation =
+            ini.getSimtkVec(section, "LEFT_TOE_STATION_LOCATION", Vec3(0));
+    auto directionWindowSize =
+            ini.getInteger(section, "DIRECTION_WINDOW_SIZE", 0);
+
+    // so parameters
+    bool solveMuscleOptimization = ini.getBoolean(section, "SOLVE_SO", true);
+    auto convergenceTolerance =
+            ini.getReal(section, "CONVERGENCE_TOLERANCE", 0.0);
+    auto memoryHistory = ini.getInteger(section, "MEMORY_HISTORY", 0);
+    auto maximumIterations = ini.getInteger(section, "MAXIMUM_ITERATIONS", 0);
+    auto objectiveExponent = ini.getInteger(section, "OBJECTIVE_EXPONENT", 0);
+    auto momentArmLibraryPath =
```

Comment:
It is strange that this passes the ctest without the absolute path.

Reply:
No clue...